nemo/collections/common/parts/rnn.py

```diff
@@ -138,6 +152,8 @@ def __init__(
         dropout: Optional[float],
         forget_gate_bias: Optional[float],
         t_max: Optional[int] = None,
+        weights_init_scale: float = 1.0,
```

Comment:
Descriptions of the new params are missing.

Reply:
Good catch.